pkg/kudoctl/packages/verifier/task/verify_basics.go

```diff
@@ -0,0 +1,29 @@
+package task
+
+import (
+	"github.com/kudobuilder/kudo/pkg/engine/task"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/verifier"
+)
+
+// ReferenceVerifier verifies tasks producing errors for tasks referenced in plans that do not exist and warnings for tasks which are not used in a plan
+type BasicVerifier struct{}
```

Comment:
More like `TasksVerifier` based on what it does?
```suggestion
type TasksVerifier struct{}
```

Reply:
That would kinda stutter on import, with `task.TasksVerifier`?
Maybe `StructureVerifier`?

Reply:
I've renamed it now to `BuildVerifier`, because that's what it does at the moment: Verifies that the task can be build